.eslintrc.json

```diff
@@ -1,10 +1,23 @@
-{
-  "extends": "bamboo",
-  "rules": {
-    "no-useless-constructor": 0,
-    "@typescript-eslint/unified-signatures": 0
-  },
-  "globals": {
-    "BigInt": true
-  }
+{	
```

Comment:
Check what actually has to be included here. The globals don't have to be probably, `ban-types` and `unified-signatures` possibly neither. same for `no-namespace`. Also possibly `no-empty-function` and with that the entire override.